jmxexporter-prometheus-grafana/assets/prometheus/jmx-exporter/confluent_ksql.yml

```diff
@@ -25,10 +26,19 @@ rules:
       id: "$3"
   # "io.confluent.ksql.metrics:type=_confluent-ksql-<cluster-id>ksql-engine-query-stats"
   # The below statement parses KSQL Cluster Name and adds a new label so that per cluster data is searchable.
-  - pattern: io.confluent.ksql.metrics<type=_confluent-ksql-(.+)ksql-engine-query-stats><>([^:]+)
+  - pattern: io.confluent.ksql.metrics<type=_confluent-ksql-(.+)_ksql-engine-query-stats><>([^:]+)
```

Comment:
I just ran the KSQLDB Server and the _ does not exist in the JMX data. Can you help revert it back ?
Eg:
io.confluent.ksql.metrics< type=_confluent-ksql-ksql-clusterksql-engine-query-stats><>ERROR-queries

Reply:
fixed, the _ got removed by accident